### PR TITLE
ok i'm done

### PR DESCRIPTION
**Pull request by jxah**

### CODE REVIEW

File: help/pics/main2.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+     <title>
+       svg learn
+     </title>
+   </head>
+   <body>
+     <p>
+       hello
+     </p>
+     <canvas hight=300 width=400>
+       hello
```
[the remark this answers is not in the document]
Comment (jxah) [reply help/pics/main2.html:13]:
```suggestion
       
```

**Review by jxah — COMMENTED** [review]
looks good

File: help/pics/main2.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<DOCUMENT>
+  <html>
+   <head>
+     <title>
+       svg learn
```
[the remark this answers is not in the document]
Comment (jxah) [reply help/pics/main2.html:5]: ok i have an idea
```suggestion
       canvas learn
```

**Review by jxah — COMMENTED** [review]
good

**Review by escambron — APPROVED** [review]

**Pull request merged**